packages/date-time-utilities/src/dateGrid/DateGrid.ts

```diff
@@ -0,0 +1,184 @@
+import { IDay, IDayGridOptions, IRestrictedDatesOptions, IAvailableDateOptions, IGridStrings } from './DateGrid.types';
+import { addDays, getDateRangeArray, compareDates, isInDateRangeArray, compareDatePart } from '../dateMath/DateMath';
+import { find } from '@uifabric/utilities';
+import { DAYS_IN_WEEK, DateRangeType, DayOfWeek } from '../dateValues/DateValues';
+
+export const getDayGrid = (options: IDayGridOptions): IDay[][] => {
```

Comment:
Can you add a basic doc comment to each of the new functions?

Reply:
Sure, would add those :)